distributed/cli/dask_scheduler.py

```diff
@@ -21,16 +22,19 @@
 
 
 @click.command()
-@click.option('--port', type=int, default=None, help="Serving port")
+@click.option('--host', type=str, default='',
+              help="URI, IP or hostname of this server")
+@click.option('--port', type=int, default=None, help="Serving port (deprecated, --host is preferred)")
```

Comment:
Agreed. Specifying both is pretty standard I think. Very useful when running multiple schedulers on one node.